starter-code/main.js

```diff
@@ -53,7 +53,187 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "10":
+            //List the name of all the products of Facebook
+            db.collection('companies').find({name: 'Facebook'}, {'products.name': 1}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result[0]);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "11":
+            //List the people that are working at Facebook right now (check relationships field)
+            db.collection('companies').find({$and: [{name: 'Facebook'}, { "relationships.is_past": true}]}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result[0].relationships);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "12":
+            //List all the companies where "david-ebersman" has worked.
+            db.collection('companies').find({"relationships.person.permalink": "david-ebersman"} , {name:1}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "13":
+            //List by name the competitors of Facebook
+            db.collection('companies').find({name:"Facebook"}, {"competitions.competitor.name": 1}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result[0].competitions);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+            case "14":
+              //Names of the companies that has "social-networking" in tag-list (be aware that the value of field is a string check regex operators)
+              db.collection('companies').find({tag_list:{$regex: /social-networking/}}, {name:1}).toArray((error, result) => {
```

Comment:
Si usas el /formato-regexp/ creo que lo puedes hacer directamente así:

```suggestion
              db.collection('companies').find({tag_list:  /social-networking/}, {name:1}).toArray((error, result) => {
```